internals/data_types.py

```diff
@@ -60,6 +60,7 @@ class StageDict(TypedDict):
   ot_is_deprecation_trial: bool
   ot_owner_email: str | None
   ot_require_approvals: bool
+  ot_setup_status: NotRequired[int]
```

Comment:
Should this be a required field and backfilled for existing Trials?

Reply:
We _could_ backfill this here for other origin trial stages, but there shouldn't be a need as long as we're querying specifically for OT stages with `OT_READY_FOR_CREATION` values, meaning that the old OT stages will be ignored.

But if we want to for data consistency purposes, it's safe to assume that any stage with a set `origin_trial_id` has already been created, so we could backfill that value this way.